### DISCUSSION TITLE
Compress on save

### DISCUSSION THREAD

**Discussion by lucapas70** [pdfarranger/pdfarranger#757] [Ideas]
It would be useful to give the possibility to compress the resulting pdf.

**Comment by jeromerobert** [comment]
This was discussed here #523 and as you can see there is almost no chance of this happening.

**Comment by lucapas70** [comment]
Sorry, I missed it. However for Windows users it would still be very convenient, because there is no floss and easy to use program to compress pdf locally. From Linux I use "Densify" instead.

**Reply by article10** [reply]
Densify uses GhostScript to reduce the filesize of .pdfs, probably by reducing image resolution.
There is a way to do the same in Windows as well easily, see my comment in the other ticket here https://github.com/pdfarranger/pdfarranger/issues/523#issuecomment-2581104814 - this maintained the OCR in my file as well.

However it would indeed be best to use something lossless like OCRMyPDF (preferably first tying the default, lossless, _--optimize 1_) (https://ocrmypdf.readthedocs.io/en/latest/cookbook.html#pdf-optimization) to compress images first (as opposed to just reducing their resolution which is what GhostScript does). 
For lossy compression it seems pdfminify can do this as well but both are not easily usable on Windows.
